### DISCUSSION TITLE
SFV renaming

### DISCUSSION THREAD

**Discussion by acceliance** [Yutaka-Sawada/MultiPar#104] [Ideas]
Please add renaming based on file checksums contained in SFV file (with no PAR/PAR2 files)

Regards

**Comment by Yutaka-Sawada** [comment]
MultiPar's SFV & MD5 formats support verification only. Because CRC-32 value in SFV isn't reliable, I don't want to implement repair future in my "SFV_MD5.exe". When you can create PAR2 file by MultiPar, you don't need SFV file. I'm not sure why you want to do it by MultiPar.

But, technically it's possible to search the same checksum file in a SFV file. If you want to rename "misnamed files" in such checksum file, you would better try a special tool for the usage. For example, I found [Python cfv](https://github.com/cfv-project/cfv). From [the manual](https://cfv.sourceforge.net/cfv.1.html) on its [original project page](https://cfv.sourceforge.net/), it supports renaming by .SFV or .MD5 files. There may be other tools of same renaming usage.
